config/clusters/opensci/common.values.yaml

```
@@ -0,0 +1,12 @@
+nfs:
+  pv:
```

This is to align with https://github.com/2i2c-org/infrastructure/pull/3666/files - it is the current default made explicit.

```suggestion
  enabled: true
  pv:
    enabled: true
```